PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalLauncher.kt

```diff
@@ -49,6 +53,33 @@ class PayPalLauncher internal constructor(private val browserSwitchClient: Brows
         }
     }
 
+    /**
+     * Launches the PayPal EditFi flow by switching to a web browser
+     *
+     * @param activity the Android Activity from which you will launch the web browser
+     * @param result a [PayPalVaultEditAuthRequest.ReadyToLaunch] received from
+     * calling [PayPalClient.createEditAuthRequest]
+     * @return [PayPalVaultEditPendingRequest] a [PayPalVaultEditPendingRequest.Started] should be stored
+     * to complete the flow upon return to app in
+     * [PayPalLauncher.handleReturnToApp],
+     * or a [PayPalVaultEditPendingRequest.Failure] with an error if the PayPal flow was unable to be
+     * launched in a browser.
+     */
+    @OptIn(ExperimentalBetaApi::class)
+    fun launch(
+        activity: ComponentActivity,
+        result: PayPalVaultEditAuthRequest.ReadyToLaunch
+    ): PayPalVaultEditPendingRequest {
+        return result.browserSwitchOptions?.let { options ->
+            when (val request = browserSwitchClient.start(activity, options)) {
+                is BrowserSwitchStartResult.Failure -> PayPalVaultEditPendingRequest.Failure(request.error)
+                is BrowserSwitchStartResult.Started -> PayPalVaultEditPendingRequest.Started(request.pendingRequest)
+            }
+        } ?: run {
+            return PayPalVaultEditPendingRequest.Failure(BraintreeException("BrowserSwitchOptions is null"))
```

Comment:
```suggestion
           PayPalVaultEditPendingRequest.Failure(BraintreeException("BrowserSwitchOptions is null"))
```

We can omit this return keyword since there is an outside return.